01_02_Combat_Rules.txt

```diff
@@ -186,6 +186,26 @@ and becomes Unconscious (and does not gain the Bleeding Out condition).
 	A target reduced to exactly 1 health by a nonlethal attack does not 
 fall Unconscious. 1 is not 0.
 
+Melee Accuracy and Guard Critical Hits:
+	See "Critical Hits" below for general Critical information. As normal, 
+a Critical Accuracy roll gives its attack +20 damage, or +30 on an Extra-critical. 
+Critical Guard rolls block all damage from any non-Extra-critical melee attack. 
+Extra-critical Guard rolls block all damage from the incoming attack. 
+
+	Melee Accuracy rolls may Critically fail in the same way as firearms. 
+Unless otherwise specified, melee weapons have a Jam range of 2. Should your 
+melee weapon "jam", it catches on something in the environment at the DM's 
+discretion, if applicable, and is stuck. Unsticking the weapon requires a 
+Weapons - Melee check as a firearm would, except that if you pass the DC by 
+at least 30 you do not provoke an Opportunity Attack. Should nothing be 
```

Comment:
I LOVE this mechanic! Simple, easy to understand and explain, and it prevents an exeption to the existing pattern! Very well done!